subsys/caf/modules/power_manager.c

```diff
@@ -162,7 +207,7 @@ static void power_down(struct k_work *work)
 
 	power_down_interval_counter++;
 	if (power_down_interval_counter <
-	    (CONFIG_CAF_POWER_MANAGER_TIMEOUT) / (POWER_DOWN_CHECK_INTERVAL_SEC)) {
+	    (power_down_timeout) / (POWER_DOWN_CHECK_INTERVAL_SEC)) {
```

Comment:
no brackets needed around power_down_timeout

Reply:
please also remove brackets from POWER_DOWN_CHECK_INTERVAL_SEC. hell knows why somebody put it there

Reply:
eh... that would be a nightmare to always use brackets on macro... people should know how to define macros and if they don't that a bug in macro definition. adding additional brackets spoils readability